sdk/tables/azure-data-tables/pom.xml

```diff
@@ -12,11 +12,10 @@ Licensed under the MIT License.
   </parent>
 
   <groupId>com.azure</groupId>
-  <artifactId>azure-cosmos-table</artifactId>
-  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-cosmos-table;current} -->
+  <artifactId>azure-data-tables</artifactId>
+  <version>1.0.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-tables;current} -->
   <name>Microsoft Azure SDK for Azure Table</name>
-  <description>This package contains the Microsoft Azure Cosmos Table client library.</description>
-  <packaging>jar</packaging>
+  <description>This package contains the Microsoft Azure Storage Table client library.</description>
```

Comment:
Should the group name be `data` or `storage`? The description says `Azure Storage Table`.